cpp/cugraph_module/algorithms/betweenness_centrality.cu

```diff
@@ -64,8 +64,8 @@ void BetweennessProc(mgp_list *args, mgp_graph *graph, mgp_result *result, mgp_m
         weight_property, kDefaultWeight);
     if (mg_graph->Empty()) return;
 
+    // TODO(gitbuda): The betweenness_centrality legacy function still works -> try to replace with latest option.
```

Comment:
Use this https://github.com/rapidsai/cugraph/blob/37b67c95a4e0f4546ee7297bb546b6aed14b01a1/cpp/include/cugraph/algorithms.hpp#L342 implementation